client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -92,11 +96,28 @@ class ProductCreate extends React.Component {
 	}
 
 	onSave = () => {
-		const { site, product, translate } = this.props;
+		const { site, product, finishedInitialSetup, translate } = this.props;
+
+		const getSuccessNotice = ( newProduct ) => {
+			if ( ! finishedInitialSetup ) {
+				return successNotice(
+					translate( '%(product)s successfully created. {{productLink}}View{{/productLink}}', {
```

Comment:
Interesting use of components for `translate()`. I don't remember seeing that before.